Sources/MongoSwift/MongoSwift.swift

```diff
@@ -8,6 +8,8 @@ private final class MongocInitializer {
 
     private init() {
         mongoc_init()
+        mongoc_log_set_handler(handleMongocLogMessage, nil)
```

Comment:
is the unstructured logging the legacy / existing mongoc log behavior? If so, I imagine we'll never expect to make use of this logging and only implement the structured logs, right?

Reply:
yes, this is the unstructured/legacy logging, and I don't think we will ever use it.

I believe Andreas is still thinking about how the old and new APIs will interact with each other. 

It's possible that if structured logging is turned on the legacy logging would be automatically turned off, in which case we could get rid of this line.